server/models/GmailCredentials.py

```diff
@@ -0,0 +1,15 @@
+from app import db
+
+from . import UserModel
+
+class GmailCredentials(db.Model):
+    __tablename__ = 'gmail_credentials'
+
+    user_email = db.Column(db.String(120), db.ForeignKey('users.email'), primary_key=True)
+    user_gmail_id = db.Column(db.String(120), unique=True, nullable=False)
+    user_gmail_credentials = db.Column(db.JSON(), nullable=False)
+    user = db.relationship('UserModel', backref='gmail_credentials', lazy=True)
```

Comment:
Is there a reason we decided to store this information as a separate model as opposed to on the user? I also noticed we had user_email and user are both from the user model, so we could just add user_gmail_id and gmail_credentials to user